hathor/conf/get_settings.py

```diff
@@ -33,6 +33,10 @@ class _SettingsMetadata(NamedTuple):
 _settings_singleton: Optional[_SettingsMetadata] = None
 
 
+def get_settings() -> Settings:
```

Comment:
I know it will cause changes in all files but I feel that `get_global_settings()` is more meaningful than just `get_settings()`.

Reply:
Ok, but may I do it in a separate PR? It'll be simpler to do it after parts 1, 2, and 3 are merged, so I don't have to update all of them separately (they are all close to getting merged). Then, it will be a simple rename action in the IDE, and a simple PR just for that.

Reply:
Ok. Thanks!